zero_downtime_migrations/backend/schema.py

```diff
@@ -240,9 +243,10 @@ def get_query_result(self, sql, params=(), row_count=False):
             return cursor.fetchone()
 
     def parse_cursor_result(self, cursor, place=0, collect_sql_value=1,):
+        result = None
         if self.collect_sql:
             result = collect_sql_value  # For sqlmigrate purpose
-        else:
+        elif cursor:
```

Comment:
this code is not clear for me. Why is cursor may not cast to "true" value?

Reply:
it is not the cursor itself but a result of cursor execution, bad naming, renamed